lib/wiki_edits.rb

```diff
@@ -284,15 +235,14 @@ def api_post(data, tokens)
       #    {"code"=>"protectedpage",
       #     "info"=>"The \"templateeditor\" right is required to edit this page",
       #     "*"=>"See https://en.wikipedia.org/w/api.php for API usage"}}
-
       if response_data['error']
-        raise StandardError.new response_data['error']['code']
+        raise StandardError.new response_data['error']['info']
```

Comment:
Agreed. We haven't done any error classes yet, so I was waiting for someone else to set the standard for how they should be done.